packages/@uppy/dashboard/src/index.js

```diff
@@ -651,13 +652,19 @@ module.exports = class Dashboard extends Plugin {
     }
   }
 
-  handleComplete = ({ failed, uploadID }) => {
+  handleComplete = ({ failed }) => {
     if (this.opts.closeAfterFinish && failed.length === 0) {
       // All uploads are done
       this.requestCloseModal()
     }
   }
 
+  _openFileEditorWhenSingleFileAdded = (file) => {
```

Comment:
not totally sure about this behaviour when adding multiple files ... it looks like this would only open the file editor for _one_ of the files?

maybe it should auto open for each file in turn? but then we'd probably need a "skip all" button in the UI in case users don't want to tweak metadata for a folder of 1000 photos.

Reply:
Yeah, if you add 20 files, you might not want to crop all of them, so this new option in my mind is more geared towards single-file use case, which is the most common. We could only open the editor if one file was added, and just show the current UI when 10 files were dropped.

Reply:
I think opening the editor if only one file was added is the second best option (the best being sequentially opening it for each file, which would require much more implementation work :p)